spec/controllers/api/v1x0/mixins/destroy_mixin_spec.rb

```diff
@@ -21,7 +20,7 @@
     end
 
     context "Sub-collection:" do
-      let!(:endpoint_1) { Endpoint.create!(:role => "a", :source => source_1, :tenant => tenant) }
+      let!(:endpoint_1) { create(:endpoint, role: "a", :source => source_1) }
```

Comment:
there is mixed syntax `a:b` and `:a => b`.
Afaik this style: `a: b` is not "approved" by rubocop (the same comment for all the similar changes in this PR)

Reply:
I understand, but this leads me to ask the following:

Do we have a project-specific style and rubocop configuration file? What's "approved" or not should be based on that configuration, not the rubocop defaults - unless that configuration accepts those defaults. 

Do we, can we , should we, run rubocop on new PRs and commits like we do in MIQ?

The rubocop configuration file should be made available to developers, maybe in the guides repo, so they can check their code before committing.

If we already have what I mentioned above, then I apologize for missing it. Otherwise, I think it would be useful to implement it.

I personally like the `a: b` notation, but if it's contrary to the officially adopted project style, I would change it.

Reply:
we have shared rubocop rules with MIQ which defines `:a => b` style. (https://github.com/RedHatInsights/sources-api/blob/master/.rubocop.yml)

The miq-bot and rubocop check should be working, we have to look why it's not working in this repo.

Reply:
@eclarizio can we get dippy-bot looking at this repo :eyes:

Reply:
@slemrmartin Thanks. Given that's the case, I the code should be changed to adhere to it.
Since I opened up the can of worms, I should put all the worms back in the can.